uint32 batch_size

uint32[]  map_crop_indices
float32[] goals_x
float32[] goals_y
float32[] goals_orient

---
float32[] costs
